### PR TITLE
[docs] Add description for multitenancy manager

### PR DESCRIPTION
**Pull request by Lada7878**
## Description

Add description for multitenancy manager.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: chore
summary: Add description for multitenancy manager.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

File: modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 1]
```diff
@@ -7,22 +7,27 @@ title: "Модуль multitenancy-manager: примеры использован
...
   * правила аудита обращения Linux-пользователей проекта к ядру;
   * сканирование запускаемых образов контейнеров на наличие известных уязвимостей (CVE).
 
+  Описание шаблона [в Github](https://github.com/deckhouse/deckhouse/blob/fb24949054e85c1aad0b63a2d0790eb1112f3334/ee/modules/160-multitenancy-manager/images/multitenancy-manager/default/secure.yaml).
+
 - `secure-with-dedicated-nodes` — включает все возможности шаблона `secure`, а также дополнительные функции:
   * определение селектора узла для всех подов в проекте: если под создан, селектор узла пода будет автоматически **заменён** на селектор узла проекта;
   * определение стандартных tolerations для всех подов в проекте: если под создан, стандартные значения tolerations **добавляются** к нему автоматически.
+  
+  Описание шаблона [в Github](https://github.com/deckhouse/deckhouse/blob/fb24949054e85c1aad0b63a2d0790eb1112f3334/ee/modules/160-multitenancy-manager/images/multitenancy-manager/default/secure-with-dedicated-nodes.yaml).
```
Comment (z9r5) [inline modules/160-multitenancy-manager/docs/USAGE_RU.md:30]:
```suggestion
  Описание шаблона [в GitHub](https://github.com/deckhouse/deckhouse/blob/main/modules/160-multitenancy-manager/images/multitenancy-manager/default/secure-with-dedicated-nodes.yaml).
```

File: modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 2]
```diff
@@ -7,22 +7,27 @@ title: "Модуль multitenancy-manager: примеры использован
...
+  
+  Описание шаблона [в Github](https://github.com/deckhouse/deckhouse/blob/fb24949054e85c1aad0b63a2d0790eb1112f3334/ee/modules/160-multitenancy-manager/images/multitenancy-manager/default/default.yaml#L2).
 
 - `secure` — включает все возможности шаблона `default`, а также дополнительные функции:
-  * настройку допустимых для проекта UID/GID;
+  * настройка допустимых для проекта UID/GID;
   * правила аудита обращения Linux-пользователей проекта к ядру;
   * сканирование запускаемых образов контейнеров на наличие известных уязвимостей (CVE).
 
+  Описание шаблона [в Github](https://github.com/deckhouse/deckhouse/blob/fb24949054e85c1aad0b63a2d0790eb1112f3334/ee/modules/160-multitenancy-manager/images/multitenancy-manager/default/secure.yaml).
```
Comment (z9r5) [inline modules/160-multitenancy-manager/docs/USAGE_RU.md:24]:
```suggestion
  Описание шаблона [в Github](https://github.com/deckhouse/deckhouse/blob/main/modules/160-multitenancy-manager/images/multitenancy-manager/default/secure.yaml).
```

File: modules/160-multitenancy-manager/docs/USAGE_RU.md [hunk 3]
```diff
@@ -7,22 +7,27 @@ title: "Модуль multitenancy-manager: примеры использован
...
 - `empty` — пустой шаблон без предопределенных ресурсов;
-
 - `default` — шаблон для базовых сценариев использования проектов:
   * ограничение ресурсов;
   * сетевая изоляция;
   * автоматические алерты и сбор логов;
   * выбор профиля безопасности;
   * настройка администраторов проекта.
+  
+  Описание шаблона [в Github](https://github.com/deckhouse/deckhouse/blob/fb24949054e85c1aad0b63a2d0790eb1112f3334/ee/modules/160-multitenancy-manager/images/multitenancy-manager/default/default.yaml#L2).
```
Comment (z9r5) [inline modules/160-multitenancy-manager/docs/USAGE_RU.md:17]:
```suggestion
  Описание шаблона [в GitHub](https://github.com/deckhouse/deckhouse/blob/main/modules/160-multitenancy-manager/images/multitenancy-manager/default/default.yaml#L2).
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Pull request closed without merging**